x-pack/test_serverless/api_integration/test_suites/common/alerting/summary_actions.ts

```diff
@@ -491,16 +490,13 @@ export default function ({ getService }: FtrProviderContext) {
       });
       ruleId = createdRule.id;
 
-      const resp = await alertingApi.helpers.waitForDocumentInIndex({
+      const resp = await alertingApi.helpers.waitForDocumentInIndexForTime({
         esClient,
         indexName: ALERT_ACTION_INDEX,
         ruleId,
         num: 2,
         sort: 'asc',
-        retryOptions: {
-          retryCount: 20,
-          retryDelay: 10_000,
-        },
+        timeout: 240_000,
```

Comment:
it is 4 minutes and mocha describe timeout is 6 min (afaik). If it really takes that long, the test will be flaky and fail with mocha timeout meaning no clear error message. 
I hardly can remember the cases, where we needed more than 2 minutes waiting, was it verified to be that long?

Reply:
Yeah the mocha timeout is 360_000 if I'm not mistaken...

Reply:
All day yesterday I played with the timeouts, even before I switched to using `tryForTime()`.
I'm not opposed to trying to bring it down, but it also has to be long enough for serverless.

Looks like there's [some history](https://github.com/elastic/kibana/pull/168225/files) of alerting api tests taking too long already as well.

I'll try to bring it down one less minute and run it through the flaky runner once more 🔹